src/utils/format.js

```diff
@@ -7,3 +8,29 @@ export let toNumber = (number, ...params) => {
   }
   return NaN
 }
+
+// replacing every character with `█`
+// preserving spaces and `|` for terms key as `name|id`
+let blank = _.memoize(_.replace(/[^ |]/gi, '█'))
```

Comment:
lets just replace alpha numeric values so we don't hard code any special knowledge here